src/main/java/com/derongan/minecraft/deeperworld/DeeperWorld.kt

```diff
@@ -34,5 +37,17 @@ class DeeperWorld : JavaPlugin() {
 
         //register command executor
         DeeperCommandExecutor
+
+        if(DeeperConfig.data.maxSafeFallingDistance != -1f){
```

Comment:
Maybe be a little more safe here. I can also imagine someone setting damage to 0 instead of safe fall distance to disable this.
```suggestion
        if(DeeperConfig.data.maxSafeFallingDistance >= 0f && DeeperConfig.data.fallingDamageMultiplier != 0f){
```

Reply:
I'll make this a `||` instead, so that only one of the two values needs to indicate that you don't want falling damage.